docs/configuration/conflict_resolution.rst

```diff
@@ -0,0 +1,35 @@
+.. include:: ../global.rst.inc
+
+
+Conflict Resolution Options
+===========================
+
+
+::
+
+   jobs:
+   - type: push
+     filesystems: ...
+     conflict_resolution:
+       initial_replication: most_recent | all | fail # default: most_recent
+
+     ...
+
+.. _conflict_resolution-initial_replication-option-send_all_snapshots:
+
+
+``initial_replication`` option
+------------------------------
+
+The ``initial_replication`` option determines which snapshots zrepl replicates if the filesystem has not been replicated before.
```

Comment:
```suggestion
The ``initial_replication`` option determines how many snapshots zrepl replicates if the filesystem has not been replicated before.
```

wdyt? might be an alternative to the comment about `all` below.

Reply:
I like that